packages/cli-kit/src/public/node/output.ts

```diff
@@ -258,7 +256,7 @@ export const clearCollectedLogs = (): void => {
  * @param content - The content to be output to the user.
  * @param logger - The logging function to use to output to the user.
  */
-export function outputInfo(content: OutputMessage, logger: Logger = consoleLog): void {
+export function outputInfo(content: OutputMessage, logger: Logger = consoleWarn): void {
```

Comment:
May I suggest a `export function output(content: outputMessage, logger: Logger = consoleLog): void {`

?

That way all the `output{log,warn,error}` would all go to stderr and we'd make it explicit which one goes to stdout?

STDOUT = scriptable output
STDERR = logging

Reply:
idk I think output is output and logging is logging. I personally don't see a problem with these two things:
- logging that goes to STDERR and requires a loglevel 
- output goes to STDOUT and doesn't require a loglevel.

STDOUT is for piping & scripts. Why would we want a loglevel in there? What am I missing?

Reply:
Because, while we don't currently support it, many programs have a `silent` log level which doesn't print anything. And I think we should ultimately support that. Keeping everything with log levels gives us more room to fine-tune logging behavior.

Reply:
Here is a good example: https://docs.npmjs.com/cli/v10/using-npm/config#loglevel

Silent is just another log level.

Reply:
Wouldn't that mean that `--json --silent` do nothing at all? There's always `&> /dev/null`? 

Whenever I'm scripting, I'm mostly annoyed by shit showing up in STDOUT and that's why I'd want `--silent`, but if it doesn't, then I don't see the point. 

One of `pnpm` or `nx` is an example offender here. If I want to run the `shopify theme language-server` with the dev build in Shopify/cli, I can only do that by running the build node package stuff because otherwise pnpm & nx are outputting crap to STDOUT that don't fit the LSP.

Same goes for `pnpm shopify theme check --json`.

Reply:
For that reason, whenever I want to test a Shopify/theme-tools bump in vim. I have a script in my $HOME/bin that is this:

```sh
#!/usr/bin/env bash

if [[ -z $DEBUG ]]; then
  node /Users/charles/src/github.com/Shopify/cli/packages/cli/bin/dev.js "$@"
else
  node --inspect-brk /Users/charles/src/github.com/Shopify/cli/packages/cli/bin/dev.js "$@"
fi
```

I need the path of the full command because the pnpm script pollutes STDOUT and breaks the LSP.

Reply:
Have you tried `pnpm --silent shopify:run ...`

`--silent` quiets pnpm

`shopify:run` skips the build step (you have to run `pnpm shopify build` first, but anyway you need to do that in your script)